build/jenkins/jobs/test/unit_test_runner.dsl

```diff
@@ -0,0 +1,38 @@
+job('unit-test-runner') {
+
+    configure { project ->
+        project / 'properties' / 'hudson.plugins.copyartifact.CopyArtifactPermissionProperty' / 'projectNameList' {
+            'string' "*rook*"
+        }
+    }
+
+    parameters {
+            stringParam('WORKSPACE', '')
+
+    }
+
+    customWorkspace('${WORKSPACE}')
+
+    throttleConcurrentBuilds {
+        maxPerNode(1)
+        maxTotal(1)
+    }
+
+    //// not yet handling test artifacts
+    // publishers {
+    //     archiveArtifacts {
+    //         pattern('results/*')
+    //         onlyIfSuccessful()
+    //     } 
+    // }
+
+    steps {
+
+        // execute test 
+        shell('(build/run make check -j${parallel})')
+
+    }
+
+    concurrentBuild false
```

Comment:
ok I'll fix #202 and we can revisit. I assume we would share a ccache across workspaces then too.